app/src/main/java/com/vero/tasky/agenda/data/local/entities/PhotoEntity.kt

```diff
@@ -0,0 +1,12 @@
+package com.vero.tasky.agenda.data.local.entities
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class PhotoEntity(
+    @PrimaryKey val id: Int? = null,
```

Comment:
I think for photos, the id needs to be generated automatically
So you'd do the opposite as I mentioned before:
```
@PrimaryKey(autogenerate = true)
```